peachjam/js/components/ListFacets.vue

```diff
@@ -99,8 +99,8 @@ export default {
         { key: 'courts', value: this.$t('Court') },
         { key: 'docTypes', value: this.$t('Document type') },
         { key: 'judges', value: this.$t('Judges') },
-        { key: 'years', value: this.$t('Year') },
-        { key: 'alphabet', value: this.$t('Alphabetical') }
+        { key: 'alphabet', value: this.$t('Alphabetical') },
```

Comment:
Since I don't think we can easily do this only for specific pages, I think we should leave it as-is for now.